.storybook/preview-head.html

```diff
@@ -1,40 +1,44 @@
 <script data-id="chakra-ui-no-flash">
   ;(function () {
-    var storageKey = "chakra-ui-color-mode"
-    var classNameDark = "chakra-ui-dark"
-    var classNameLight = "chakra-ui-light"
+    window.addEventListener("DOMContentLoaded", function (e) {
```

Comment:
I have only added this line of code to check if the DOMContent is loaded before running the below code.
Rest of the code being formatted by prettier.